src/instrumenter/cha.ts

```diff
@@ -197,13 +194,18 @@ export function chaDFS<T>(cha: CHA<T>, cb: (node: T) => void): void {
         }
 
         seen.add(cur);
+
         cb(cur);
 
         const children = cha.children.get(cur);
+
         assert(
             children !== undefined,
-            `Node ${pp(cur)} is missing from children map in ${ppCHA(cha)}`
+            "Node {0} is missing from children map in {1}",
+            cur,
+            ppCHA(cha)
```

Comment:
Same here - `ppCHA` is fairly expensive. Maybe move assert inside of if?

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/94/commits/d4ed33eadbc26a96e9b7719cff33f967783af572